# Represents a maliput::api::JunctionId.
string id
